src/coreclr/nativeaot/System.Private.CoreLib/src/System/Delegate.cs

```diff
@@ -443,27 +443,8 @@ private string GetTargetMethodsDescriptionForDebugger()
             }
             else
             {
-                RuntimeTypeHandle typeOfFirstParameterIfInstanceDelegate;
-                IntPtr functionPointer = GetFunctionPointer(out typeOfFirstParameterIfInstanceDelegate, out bool _, out bool _);
-                if (!FunctionPointerOps.IsGenericMethodPointer(functionPointer))
-                {
-                    return DebuggerFunctionPointerFormattingHook(functionPointer, typeOfFirstParameterIfInstanceDelegate);
-                }
-                else
-                {
-                    unsafe
-                    {
-                        GenericMethodDescriptor* pointerDef = FunctionPointerOps.ConvertToGenericDescriptor(functionPointer);
-                        return DebuggerFunctionPointerFormattingHook(pointerDef->InstantiationArgument, typeOfFirstParameterIfInstanceDelegate);
```

Comment:
Why is it ok to delete all of this?

Reply:
The whole GetTargetMethodsDescriptionForDebugger method (not just this section) and the DebuggerDisplayAttribute on Delegate that references the method can be deleted.

They were used by the .NET Native debugger. They have no use right now. We can restore from history if it becomes relevant again. Debuggers display the targets in less luxurious way in NativeAOT right now.